### DISCUSSION TITLE
Barb reporting

### DISCUSSION THREAD

**Discussion by mirakhany** [isid0re/SoloLeveling#313] [Q&A]
Hi. thank you for Barb update.
a few reporting with my lv87 Barb(Leveling Build)
Sometimes he never stop crying when he moving.
and not enough inventory space(charms, too many potion buffers) for pick items = no money
and he looks need more stats(str, dex) for gears.

personal question.(Im gonna very happy if solve this)
How can I add aaa.nip in config.js?

**Comment by mirakhany** [comment]
Hi. my Barb made PhaseBlade Grief. but he has not enough dex. also he won't start respec two.
and he sold Grief.
please check this.

**Comment by isid0re** [comment]
Thanks for posting, I'm going thru all the builds to fix stat problems like you have mentioned.

**Comment by isid0re** [comment]
I've got a fix for the stutter walking and shouting coming shortly as well as updated stats and builds. Additionally I missed your question regarding adding aaa.nip. 

to do so just add the following to your class.sololeveling.js config file on line 130.
`Config.PickitFiles.push("aaa.nip");`

-isi

**Reply by mirakhany** [reply]
Hi. thanks you for update.
So far so good. My Barb reached final build. so I can't test leveling build inventory issue anymore.
if baal's 5th wave monster has physical immune, he doesn't attack and never finish run.(maybe other characters have same issue with immune)
and some characters(classes) too far from baal's wave starting(spawn) point. so they never encounter. just standing.
please check these.

and thank you for your answer.
but I want to know insert "Config.PickitFiles.push("aaa.nip");" into config.js not class.sololeveling.js. if possible.

**Reply by isid0re** [reply]
Class.sololeveling.js is the character config settings file. 

For example,
Sorceress.sololeveling.js in the libs/config folder.
